.env.example

```
@@ -1,10 +1,21 @@
-# Database Configuration
+프롲# Database Configuration
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The 프롲# Database Configuration key has incorrect delimiter

(IncorrectDelimiter)